<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>385</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>370</width>
    <height>385</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>370</width>
    <height>385</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SimpleGUI</string>
  </property>
  <widget class="QLineEdit" name="txt1">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>30</y>
     <width>201</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>90</y>
     <width>201</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Value 1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Value 2</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_out">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>210</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>320</y>
     <width>170</width>
     <height>29</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QPushButton" name="btn_exit">
    <property name="text">
     <string>&amp;Exit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_cal">
    <property name="text">
     <string>&amp;Calculate</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
